src/AsyncTCP.h

```diff
@@ -26,8 +26,10 @@
 #include "sdkconfig.h"
 #include <functional>
 #include <string>
+#if ASYNC_TCP_SSL_ENABLED
 #include <ssl_client.h>
 #include "tcp_mbedtls.h"
+#endif
```

Comment:
This isn't going to work... Have you tried compiling with ASYNC_TCP_SSL_ENABLED=0 ? There is code that depends on the definitions in the header file, so if you remove the header file you also have to remove the code that depends on it.

Reply:
Never mind, I forgot that he other code is already #ifdef'd, I just forgot to also exclude the #include I gather.

Reply:
compiles fine when set to 0, yes.

Reply:
compiles fine when set to 0, yes.